weblate/templates/snippets/project/billing-libre-trial.html

```diff
@@ -8,10 +8,10 @@
   {% if user_can_view_billing %}
     {# project admin can request approval #}
     <a href="{{ object.billing.get_absolute_url }}" class="btn btn-primary pull-right flip">{% trans "Request approval for Libre hosting" %}</a>
-    {% trans "This project is in the trial period, and is publicly accessible. You can lock it and post an announcement in case you want to avoid contributions until you finish the setup." %}
+    {% trans "This project is in the trial period and is publicly accessible. You can post an announcement and lock the project to avoid contributions until you finish the setup." %}
```

Comment:
This is just worse. It was grim before, and now it doesn't make sense in a myriad of ways.

Reply:
You’re saying it’s worse and proposing changes that go away from how it really works and what we as a team need to tell the admins. I am sorry, I can’t accept them.

Reply:
I proposed the changes you made. https://github.com/WeblateOrg/weblate/discussions/6790#discussioncomment-1681159
However, you shoehorned it into a message written by nijel.
What that does is focus on locking and posting announcements, when that should be the exception.
It further doesn't say what the announcement should be.

The changes I proposed are for users, not admins. https://github.com/WeblateOrg/weblate/pull/6855/files

I didn't imagine you would care in the slightest about how anything is worse, because I thought you did nothing about
https://github.com/WeblateOrg/weblate/pull/6846#issuecomment-976591434

What I would do is ask what "a myriad of ways" means.
Otherwise I have no idea of how "It will be solved" https://github.com/WeblateOrg/weblate/pull/6855#issuecomment-978032738 is going to be anywhere close.

All that to not solve a problem one(?) person already had, in 700+ projects.

Edit: Happy to see something happened to "Ask the maintainer if it is ready yet". I overlooked that.